firmware/common/portapack_persistent_memory.cpp

```diff
@@ -923,5 +931,122 @@ size_t data_size() {
     return sizeof(data_t);
 }
 
+bool debug_dump() {
+    ui::Painter painter{};
+    std::string debug_dir = "DEBUG";
+    std::filesystem::path filename{};
+    File pmem_dump_file{};
+    // create new dump file name and DEBUG directory
+    make_new_directory(debug_dir);
+    filename = next_filename_matching_pattern(debug_dir + "/DEBUG_DUMP_????.*");
```

Comment:
You can just do `/DEBUG/dump_????.txt` or whatever direction here. No need to use the * and then replace extension.

Reply:
Done thanks